api/v1beta1/webhook_suite_test.go

```diff
@@ -93,6 +93,9 @@ var _ = BeforeSuite(func() {
 	err = (&OpenStackNetAttachment{}).SetupWebhookWithManager(mgr)
 	Expect(err).NotTo(HaveOccurred())
 
+err = (&OpenStackDeploy{}).SetupWebhookWithManager(mgr)
```

Comment:
nit: indent

Reply:
This isn't needed. I'll just remove in the latest patch.